src/components/DiffView/index.tsx

```diff
@@ -131,13 +135,18 @@ export class DiffViewBase extends React.Component<Props> {
     _document: document,
     _getDiffAnchors: getDiffAnchors,
     _getRelativeDiffAnchor: getRelativeDiffAnchor,
+    _sendPerfTiming: sendPerfTiming,
     _slowLoadingCharCount: SLOW_LOADING_CHAR_COUNT,
     _tokenize: tokenize,
     _trimmedCharCount: TRIMMED_CHAR_COUNT,
     enableCommenting: process.env.REACT_APP_ENABLE_COMMENTING === 'true',
     viewType: 'unified',
   };
 
+  onRenderCallback = (id: string, phase: string, actualDuration: number) => {
```

Comment:
```suggestion
  onRenderProfiler = (id: string, phase: string, actualDuration: number) => {
```

Reply:
We should likely add a link to the docs: https://github.com/reactjs/rfcs/blob/master/text/0051-profiler.md above this method.